src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -112,16 +113,14 @@ private String encrypt(String value) {
         return null;
     }
 
-    private String decrypt(String value) {
+    private String decrypt(final String key, String value) {
         try {
             return sHelper.decrypt(value);
         } catch (Exception e) {
             Logger.e(TAG, "Decryption failure", "", ADALError.ENCRYPTION_FAILED, e);
             if (!StringExtensions.IsNullOrBlank(value)) {
```

Comment:
do you need to check it for null? remove anyway, there shouldn't be empty records in cache...